README.md

```diff
@@ -120,15 +162,21 @@ And we wish to rollback production to `1.0.4` then:
 npm dist-tag add my-module@1.0.4 prod
 ```
 
-This will trigger a build of `my-module@1.0.4` (since it's dependencies may have changed since 1.0.4 was latest in production) and a build of all modules that depend on `my-module`.
+This will trigger a build of `my-module@1.0.4` (since it's dependencies may
+have changed since 1.0.4 was latest in production) and a build of all modules
+that depend on `my-module`.
 
 ### Auto-update of builds
 
 > Stability: 1 – **Unstable**
 
-The first (and most important) point developers need to be aware of is that builds from the Warehouse are always on latest version of private dependencies tagged with that particular environment.
+The first (and most important) point developers need to be aware of is that
+builds from the Warehouse are always on latest version of private dependencies
```

Comment:
`always` is a strong word. it doesn't constantly update existing builds, you only get latest semver matching versions on new builds, not for existing builds.

Reply:
again, can be addressed later since you're just updating formatting ;)